README.md

```diff
@@ -13,12 +19,36 @@
 - create, modify and delete environment variables
 - convert environment variables to a secret or configmap
 - modify the DDL part of the virtual database
+Red Hat Integration - Data Virtualization Tooling enables you to create a Virtual Database definition file (VDB) and to visualize and manipulate its contents inside [VS Code](https://code.visualstudio.com/).
```

Comment:
I think, there need to be empty line inserted

Reply:
or it could be above list?

Reply:
actually I forgot to remove that line.